app/scripts/main.js

```diff
@@ -0,0 +1,28 @@
+window.onload = () => {
+  document.getElementById('try-params').addEventListener('submit', (form) => {
+    form.preventDefault();
+
+    /**
+     * sends a request to the specified url from a form. this will change the window location.
+     * @param {string} path the path to send the post request to
+     * @param {object} params the paramiters to add to the url
+     * @param {string} [method=post] the method to use on the form
+     */
```

Comment:
again don't think these comments are required here. I think the function signature gives us all this detail